sdk/tests/client/input_selection/burn.rs

```diff
@@ -735,21 +735,21 @@ fn burn_foundries_present() {
         Foundry(
             1_000_000,
             alias_id_1,
-            0,
+            1,
             SimpleTokenScheme::new(U256::from(0), U256::from(0), U256::from(10)).unwrap(),
             None,
         ),
         Foundry(
             1_000_000,
             alias_id_1,
-            1,
+            2,
             SimpleTokenScheme::new(U256::from(0), U256::from(0), U256::from(10)).unwrap(),
             None,
         ),
         Alias(
             1_000_000,
             alias_id_1,
-            0,
+            2,
```

Comment:
Is this checked? I changed it back to 0 locally and it passed

Reply:
It doesn't matter what we have there, can also undo

Reply:
Why doesn't it matter?

Reply:
Because we don't provide it in the outputs

Reply:
Ok, still keep 2, it makes sense. Was just wondering why it didn't fail.